routes/web.php

```diff
@@ -1,4 +1,4 @@
-<?php
```

Comment:
Please cleanup this file - this should only reflect changes that are necessary for your PR to work. It looks like there are many other unnecessary changes here (extra lines added, etc). Additionally, all php files should have the `<?php` directive at the top, so that does need to be restored.